module/chain.go

```diff
@@ -47,6 +47,6 @@ func (c *ethChain) CanonicalChainID(ctx context.Context) (uint64, error) {
 	return chainID.Uint64(), nil
 }
 
-func (c *ethChain) GetProof(address common.Address, storageKeys [][]byte, blockNumber *big.Int) (*client.StateProof, error) {
-	return c.Client().GetProof(address, storageKeys, blockNumber)
+func (c *ethChain) GetProof(ctx context.Context, address common.Address, storageKeys [][]byte, blockNumber *big.Int) (*client.StateProof, error) {
+	return c.Client().GetProof(ctx, address, storageKeys, blockNumber)
```

Comment:
The signature has changed in the PR https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/64#discussion_r1978699763.